gapis/config/config.go

```diff
@@ -27,8 +27,9 @@ const (
 	// Logs all mappings at the end of the replay from original trace
 	// handles to replay client handles (if handles are reused in the trace
 	// it will only print the last mapping).  Only works for Vulkan.
-	LogMappingsToFile      = false
-	LogTransformsToFile    = false
-	LogTransformsToCapture = false
-	SeparateMutateStates   = false
+	LogMappingsToFile          = false
+	LogTransformsToFile        = false
+	LogTransformsToCapture     = false
+	SeparateMutateStates       = false
+	MutateDuringReconstruction = false
```

Comment:
Yes, it mutates the state, but this is just part of the work to verify that the commands correctly reconstruct the state. So how's about `CheckRebuiltStateMatches` or something, as this better describes the feature and also includes the costly call to `compare.Compare`?

Reply:
The idea was for Vulkan to also use this setting. @AWoloszyn, is that still something that has a chance of happening? any input on the name?

Reply:
It can be done (technically) but it would be quite a non-trivial change to make it work. It is however something we should strive for. 
`CheckRebuiltStateMatches` is fine with me.

Reply:
Done. PTAL